utils/das_data_sorter.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd 
+import argparse 
+from math import *
+
+#accepts terminal commands
+parser = argparse.ArgumentParser()
+parser.add_argument("--file", help="Input CSV file", action="store")
+parser.add_argument("--output", help="Returns the filtered data", default="filtered_data.csv", action="store")
+args = parser.parse_args()
+
+#initialising data
+data = pd.read_csv(args.file)
+time = data['time']/1000
+gps_course = data['gps_course']
+gps_speed = data['gps_speed']
+tempC = data['thermoC']
```

Comment:
In python we generally use `snake_case` for variable and function names and `PascalCase` for class names (although this isn't always consistent). See [PEP8](https://www.python.org/dev/peps/pep-0008/) for the official style guide. It's not so important to stick to this 100%, just as long as you're always consistent with your code style and formatting

Reply:
I understand what you're saying and it is a good practice to follow all the time

I kept the variables as 'tempC' and 'tempF', because I (and others who would want to adapt the script) didn't want to write an underscore everytime for just the letter C or F. The variable names were similar to the header names from the csv, so I also want to keep that consistency.